test/cbmc/proofs/arraySearch/Makefile

```diff
@@ -0,0 +1,23 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+PROOF_UID=arraySearch
```

Comment:
Should `EXPENSIVE = true` be set considering that this proof takes upwards of 10GB per the graphs displayed on https://deh3ggah7odfx.cloudfront.net/final/ee34e1b2-bdad-4777-94cb-e02f32a590e1/index.html ?

Reply:
Implemented as suggested.